src/main.rs

```diff
@@ -179,6 +179,7 @@ fn main() {
     let service_name = matches.value_of("service_name").unwrap_or("");
     let service_number = matches.value_of("service_number").unwrap_or("");
     let service_job_number = matches.value_of("service_job_number").unwrap_or("");
+    let branch = matches.value_of("branch").unwrap_or("master");
```

Comment:
You'll need to add the option to the clap App definition above too

Reply:
Ah, I forgot we already have a `branch` option, but it is to enable branch coverage parsing!

So we need to change the name of this option, you can call it vcs_branch

Reply:
oh, my bad. Give me 5 :)

Reply:
Just pushed the update. Not sure if you have a preference on the variable name too but I changed the var name to vcs_branch, not just the input string.